cherrypy/lib/cpstats.py

```diff
@@ -308,6 +308,14 @@ def _get_threading_ident():
     return threading._get_ident()
 
 
+def _get_response_status(resp):
+    try:
```

Comment:
I prefer the original syntax:

```
response_status = getattr(resp, 'output_status', None) or resp.status
```

It achieves the same thing in one line instead of four, but more importantly, it causes `response_status` to be assigned in exactly one place.

Since this PR isn't about addressing the code style, better to focus on changing just the one thing.

Reply:
Let's do `response_status = getattr(resp, 'output_status', resp.status)` then.

Reply:
If `resp.output_status` can be `None`, then you want `getattr(resp, 'output_status') or resp.status`.

Reply:
I guess let's try webknjaz' suggestion and see if that works.